cmd/importaccount.go

```diff
@@ -46,6 +46,7 @@ func createImportAccountCmd() *cobra.Command {
 	cmd.Flags().BoolVarP(&params.skip, "skip", "", false, "skip validation issues, they can be edited out prior to push")
 	cmd.Flags().BoolVarP(&params.overwrite, "overwrite", "", false, "overwrite existing account")
 	cmd.Flags().StringVarP(&params.file, "file", "j", "", "account jwt to import")
+	cmd.Flags().BoolVarP(&params.force, "force", "", false, "import account signed by different operator")
```

Comment:
I think this is what the migrate command does.

Reply:
the force there does smth else. account id is never checked.